samples/drivers/uart/single_wire/src/main.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021 Jonathan Hahn
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include "kernel.h"
+#include <device.h>
+#include <devicetree.h>
+#include <drivers/uart.h>
+
+#define USART2_NODE DT_NODELABEL(usart2)
+#define UART4_NODE DT_NODELABEL(uart4)
+
+const struct device *usart2 = DEVICE_DT_GET(USART2_NODE);
+const struct device *uart4 = DEVICE_DT_GET(UART4_NODE);
```

Comment:
this makes sample board specific, can be improved (e.g. using aliases?)

Reply:
done